root/static/scripts/edit/URLCleanup.js

```diff
@@ -424,9 +424,9 @@ const CLEANUPS = {
             if (prefix === 'album/mw') {
               return {
                 error: l(
-                  `Allmusic album links should be added to release groups.
-                   To find the appropriate release link for this release,
-                   please check the Releases tab for the Allmusic album.`,
+                  `Allmusic “album” links should be added to release groups.
+                   To find the appropriate Allmusic link for this release,
+                   please check the “Releases” tab for the Allmusic “album”.`,
```

Comment:
I was thinking whether to say "/album" links instead at first, and if you're going to quote the first one I'd probably prefer the slash to be added there. I think the second album was fine as it was, but otherwise I'd prefer `check the “Releases” tab of the Allmusic page.` or something like that. The quotation otherwise makes it sound weird to me, like "they call them albums but they're wrong".

Reply:
* Similarly as below, I propose `<code>/album</code>` for the first one,
* For the second one, maybe we can directly use the link? 
  That would be `from {album_url}.` with `album_url: (<a href={url}>url</a>),`.

Reply:
Maybe `from {album_url|your link}`? Rather than reprinting the URL? But I like the idea!

Reply:
* Fixed at https://github.com/metabrainz/musicbrainz-server/pull/1536/commits/2683b3e6da835c600ffb2dabda28a13eb0853977#diff-87814c26df342be7e4c2a85531bbd47cL427-R435
* Fixed with https://github.com/metabrainz/musicbrainz-server/pull/1536/commits/b8bd4d62456eb5a2e0ba6df74641e66c5ad13c97